apps/demo/src/app/app.component.html

```diff
@@ -111,10 +111,10 @@ <h2>Next Steps</h2>
     <summary>Add UI library</summary>
     <pre>
   # Generate UI lib
-  nx g @nx/angular:lib ui
+  nx g &#64;nx/angular:lib ui
```

Comment:
This is on purpose, I guess?

Reply:
yes, the `@` was causing compilation issues :) see https://github.com/geonetwork/geonetwork-ui/pull/1070/commits/c831ae859935512e9e458dc867b42001e1588c3d